applications/vanilla/controllers/class.postcontroller.php

```diff
@@ -241,13 +241,11 @@ public function discussion($CategoryUrlCode = '') {
                     }
                 }
 
-                // Make sure that the title will not be invisible after rendering
-                if (unicodeRegexSupport()) {
```

Comment:
No need. There is no equivalent to /\w/u if unicode is not supported.
And in that case the u modifier will just be ignored.